app/views/services/govbox/pricing.html.erb

```diff
@@ -37,13 +37,13 @@
       </ul>
 
       <h1 id= "api" class="mt-5">Cenník GovBox API</h1>
-      <p class="mb-5"><small>Platný od 20.04.2022</small></p>
+      <p class="mb-5"><small>Platný od 14.06.2023</small></p>
 
       <p>Poplatok za poskytnutie služby GovBox API (odmena) sa stanovuje na základe počtu Používateľom stiahnutých a odoslaných elektronických správ (definovaných v <%= link_to 'Podmienkach používania služby GovBox API', terms_api_services_govbox_index_path %>) za podmienok uvedených v Zmluve a v Podmienkach používania služby GovBox API, pričom:</p>
       <ul>
-        <li>prvý mesiac nasledujúci po mesiaci v ktorom Zmluva nadobudla účinnosť je služba GovBox API poskytovaná bezplatne; a v nasledujúcich mesiacoch je služba účtovaná na základe kategórie, do ktorej spadá, a to podľa počtu stiahnutých a odoslaných elektronických správ nasledovne:
+        <li>prvý mesiac nasledujúci po mesiaci v ktorom Zmluva nadobudla účinnosť je služba GovBox API poskytovaná bezplatne; a v nasledujúcich mesiacoch je služba účtovaná na základe kategórie, do ktorej spadá, a to podľa počtu stiahnutých elektronických správ nasledovne:
           <ol type="A">
-            <li><span>v prípade ak v kalendárnom mesiaci nebude stiahnutá alebo odoslaná žiadna elektronická správa, tak je služba za tento mesiac poskytovaná bezodplatne,</span></li>
+            <li><span>v prípade ak v kalendárnom mesiaci nebude stiahnutá žiadna elektronická správa, tak je služba za tento mesiac poskytovaná bezodplatne,</span></li>
```

Comment:
Toto sa mi fakt nezda, mi uctujeme aj za odoslane spravy, cize nevidim dovod to davat prec. To je asi chyba v tom zneni VOP